pages/index.vue

```diff
@@ -24,6 +24,7 @@
           class="hero-section__join-button"
           discord
           href="https://discord.gg/tN5Wcab"
+          target="_blank"
```

Comment:
add `rel="noopener"` that the page from which you are redirected can't track the website from which you are